packer.mk

```diff
@@ -37,6 +37,21 @@ else
 	@packer build -var-file=vhdbuilder/packer/settings.json vhdbuilder/packer/vhd-image-builder-mariner.json
 endif
 else ifeq (${OS_VERSION},V2)
+ifeq (${ARCHITECTURE}, ARM64)
+ifeq (${MODE},gen2Mode)
+	@echo "${MODE}: Building with Hyper-v generation 2 ARM64 VM"
+	@rm -f ./vhdbuilder/packer/packer
+	@curl -fsSL https://packerbinary.blob.core.windows.net/bin/packer -o ./vhdbuilder/packer/packer
```

Comment:
We don't need to use my private packer builds to workaround an ARM64 os disk capturing issue. Please see my recent PR https://github.com/Azure/AgentBaker/pull/1956 to make corresponding changes

Reply:
Addressed with a git commit --amend, thank you!